nrf-hal-common/src/twim.rs

```diff
@@ -86,6 +86,16 @@ where
         Twim(twim)
     }
 
+    /// Enable the instance.
```

Comment:
```suggestion
    /// Re-enable the instance after it was previously disabled.
```

Reply:
Maybe `enable` should be moved below `disable`, since you always call them in that order.